contracts/StablecoinConverter.sol

```diff
@@ -224,23 +226,81 @@ contract StablecoinConverter is EpochTokenLocker {
         tokenConservation.checkTokenConservation();
         documentTrades(batchIndex, owners, orderIds, volumes, tokenIdsForPrice);
     }
+    /**
+     * Public View Methods
+     */
+
+    /** @dev View returning ID of listed tokens
+      * @param addr address of listed token.
+      * @return tokenId as stored, via BiMap, within the contract.
```

Comment:
I think the fact that we are using a BiMap is an implementation detail that we don't need to document.

Reply:
```suggestion
      * @return tokenId as stored within the contract.
```